src/config/product.openshiftContainer.js

```diff
@@ -292,20 +286,36 @@ const config = {
         }),
       isSort: true,
       isWrap: true,
-      width: 20
+      width: 10
+    },
+    {
+      metric: RHSM_API_PATH_METRIC_TYPES.CORES,
```

Comment:
regardless of whether we transform the api response to have `Cores` there is no metric id sorting on the subscriptions api

required: 
- change this back to total_capacity
- or disable the sorting
- or do both